tcp.py

```diff
@@ -105,6 +105,43 @@ def send_message(self, msg_str=None):
         # if msg_str is none, just send the return data?
         pass
 
-    # This method returns the data as well as updates
-    # 'return_data_str', so having a return seems
-    # uneccesary
\ No newline at end of file
+
+'''
+This placement emulates the file structure of the labview code, although I 
+don't think this is the best place for these functions

```

Comment:
I've placed these functions here because that's where they are in the labview file structure but I'm definitely unsure as to whether this is the best place for them
@c-poole you mentioned having worked on some data encoding from labview to xml in the past? Could you enlighten us on how that aught to be done?